substrate/primitives/api/test/tests/decl_and_impl.rs

```diff
@@ -143,6 +146,18 @@ impl_runtime_apis! {
 			unimplemented!()
 		}
 	}
+
+	pub use ext::*;
+}
+
+#[sp_api::impl_runtime_apis_ext]
+mod ext {
+	use super::*;
+	impl super::Example<Block> for Runtime {
+		fn same_name() -> String {
+			"example".to_string()
+		}
+	}
 }
```

Comment:
Having 2 external implementation would be even better for testing. like ext1 and ext2.